components/usage/pkg/db/workspace_instance_usage.go

```diff
@@ -91,3 +91,14 @@ func ListUsage(ctx context.Context, conn *gorm.DB, attributionId AttributionID,
 }
 
 type UsageReport []WorkspaceInstanceUsage
+
+func (r UsageReport) FilterToAttributionID(attributionID AttributionID) UsageReport {
```

Comment:
Do you think it's worth adding a test for this method (as it is exported by the `db` package)?

Reply:
Good shout. Will add it.

I've also rebased and used the new `contentservice.UsageReport` struct

Reply:
Added tests